README.md

```diff
@@ -166,6 +170,11 @@ the cluster id, and keypair:
 --r53-zone my.hosted.domain --rhsm-user my_redhat_user --rhsm-pass my_redhat_pass
 ```
 
+More complex version
+```
+./run.py --no-confirm --cluster-id openshift --num-nodes 1 --keypair default --r53-zone alberttwong.com --rhsm-user alwong@redhat.com --rhsm-pass n0n34u --region us-west-1 --ami ami-d1315fb1 --run-smoke-tests --num-smoke-test-users 5 
+```
```

Comment:
* What's wrong with the example?
* Running `run.py` with no options will prompt you for everything you need
* If the defaults are not sensible, then please submit a PR / suggest changes against the defaults.

Your "more complex version" is only required if you don't want to accept any defaults. Not specifying a user/password will force you to be prompted. It's not required. Smoke tests are optional. Region and AMI are optional unless you don't want to use a default, but they are documented, etc etc.

I'm all for accepting changes but if you are going to use a "more complex version" then you would likely want to include something like all sensible options, not just the few sensible options that might make sense outside of the defaults, no?